tsconfig.json

```diff
@@ -6,17 +6,20 @@
     "moduleResolution": "Node",
     "sourceMap": true,
     "strict": true,
-    "target": "ES2017",
-    "typeRoots": [
-      "./node_modules/@types",
-    ],
+    "target": "ES2021",
+    "lib": ["ES2021", "DOM"],
```

Comment:
Extension is `"lib": ["DOM", "es2023"],`, so this is certainly fine; it was the minimum version bump to use some of the features I was already using within Extension (before I copied my changes over to here).